record-and-playback/core/lib/recordandplayback/edl/audio.rb

```diff
@@ -22,8 +22,11 @@ module EDL
     module Audio
       FFMPEG_AEVALSRC = "aevalsrc=s=48000:c=stereo:exprs=0|0"
       FFMPEG_AFORMAT = "aformat=sample_fmts=s16:sample_rates=48000:channel_layouts=stereo"
+      FFMPEG_AFORMAT_SCREENSHARE = "aresample=async=1000,aformat=sample_fmts=s16:sample_rates=48000:channel_layouts=stereo"
```

Comment:
Thinking about it a bit, we should probably apply the `aresample=async=XXX` filter to all of the audio inputs. It won't hurt anything and who knows, it might help improve audio sync in a few cases?

Reply:
> it might help improve audio sync in a few cases

Yeah. While I didn't implement this, I suggested the resample to try and tackle desync issues @germanocaumo was dealing with on streams riddled with gaps due to simulated packet drops. It did improve it by a large margin. So might be a good idea to put it in the other inputs as well yes.

I'd do it in a separate PR if possible?

Reply:
I'd like this PR to use the same audio format for all audio, there's no reason to special-case the screenshare audio.